candidates/management/commands/candidates_create_csv.py

```diff
@@ -6,13 +6,46 @@
 from tempfile import NamedTemporaryFile
 
 from django.core.management.base import BaseCommand, CommandError
-from django.db.models import F
+from django.db import reset_queries
 
 from candidates.csv_helpers import list_to_csv
 from candidates.models import PersonExtra
+from candidates.models.fields import get_complex_popolo_fields
 from elections.models import Election
 
 
+FETCH_AT_A_TIME = 1000
+
+
+def queryset_iterator(qs, complex_popolo_fields):
+    # To save building up a huge list of queries when DEBUG = True,
+    # call reset_queries:
+    reset_queries()
```

Comment:
I don't think it matters much, since the difference is between 15 or so queries being in `connection.queries` or not - I was really worried about thousands ending up there if there's no reset at all.